### PR TITLE
[TM-978] Site profile monitoring map

### PR DESCRIPTION
**Pull request by cesarLima1**
https://gfw.atlassian.net/browse/TM-978
depends on: https://github.com/wri/wri-terramatch-api/pull/293

### CODE REVIEW

File: src/components/elements/MapPolygonPanel/AttributeInformation.tsx [hunk 1]
```diff
@@ -1,75 +1,167 @@
...
   {
     title: "Partial",
-    value: 2
+    value: "Partial"
   },
   {
-    title: "Full",
-    value: 3
+    title: "Full Coverage",
+    value: "Full Coverage"
```
Comment (roguenet) [inline src/components/elements/MapPolygonPanel/AttributeInformation.tsx:73]: If I'm reading this correctly, the titles for these drop down options aren't getting translated. It's a little messy, but the first thing that comes to mind for me to manage that would be create a hook that maps these option lists in a `useMemo()` and uses `useT()` to translate the titles. I'm open to other options, as long as the text that gets displayed to the user gets passed through transifex somehow.

File: src/pages/site/[uuid]/tabs/Overview.tsx [hunk 2]
```diff
@@ -137,200 +173,201 @@ const SiteOverviewTab = ({ site }: SiteOverviewTabProps) => {
...
-                <Dropdown
-                  containerClassName="w-full"
-                  placeholder={t("All Polygons")}
-                  options={[
+        </PageRow>
+        <PageRow>
+          <PageColumn>
+            <PageCard
+              title={t("Site Area")}
+              tooltip="Lorem ipsum dolor sit amet, urna neque viverra justo nec ultrices dui sapien eget mi proin sed libero."
```
Comment (roguenet) [inline src/pages/site/[uuid]/tabs/Overview.tsx:223]: Please make sure we're not including any mocked / test data in merges to staging. Also, if we have real text to bring in for this tooltip, it'll need to be wrapped in `t()`

File: src/pages/site/[uuid]/tabs/Overview.tsx [hunk 3]
```diff
@@ -137,200 +173,201 @@ const SiteOverviewTab = ({ site }: SiteOverviewTabProps) => {
...
+              </div>
+              <SiteArea sites={site} setEditPolygon={setEditPolygon} editPolygon={editPolygon} />
+            </PageCard>
+          </PageColumn>
+        </PageRow>
+        <PageRow>
+          <PageColumn>
+            <PageCard
+              title={t("Project Monitoring")}
+              tooltip="Lorem ipsum dolor sit amet, urna neque viverra justo nec ultrices dui sapien eget mi proin sed libero."
```
Comment (roguenet) [inline src/pages/site/[uuid]/tabs/Overview.tsx:267]: Same comment here as on the tooltip above.

File: src/pages/site/[uuid]/tabs/Overview.tsx [hunk 4]
```diff
@@ -137,200 +173,201 @@ const SiteOverviewTab = ({ site }: SiteOverviewTabProps) => {
...
+            >
+              <div className="flex items-center justify-between text-darkCustom">
+                <Text variant="text-14-light" className="w-[65%]">
+                  Select all or specific sites to view remote sensing analytics such as tree counts, NDVI, and other
```
Comment (roguenet) [inline src/pages/site/[uuid]/tabs/Overview.tsx:271]: Wrap all free text in this component in `t()`. I see this and some properties (like placeholders and options titles) below.

File: src/pages/site/[uuid]/tabs/Overview.tsx [hunk 5]
```diff
@@ -137,200 +173,201 @@ const SiteOverviewTab = ({ site }: SiteOverviewTabProps) => {
...
+                    placeholder="All Polygons"
+                    options={[
+                      {
+                        title: "All Polygons",
+                        value: 1
+                      },
+                      {
+                        title: "All Polygons2",
+                        value: 2
+                      }
```
Comment (roguenet) [inline src/pages/site/[uuid]/tabs/Overview.tsx:286]: Is this mocked data? These options seem suspicious.

File: src/components/elements/MapPolygonPanel/AttributeInformation.tsx (hunk 1 above)
Comment (egrojMonroy) [reply src/components/elements/MapPolygonPanel/AttributeInformation.tsx:73]: done

File: src/pages/site/[uuid]/tabs/Overview.tsx (hunk 2 above)
Comment (egrojMonroy) [reply src/pages/site/[uuid]/tabs/Overview.tsx:223]: tooltip removed

File: src/pages/site/[uuid]/tabs/Overview.tsx (hunk 3 above)
Comment (egrojMonroy) [reply src/pages/site/[uuid]/tabs/Overview.tsx:267]: remove tooltip

File: src/pages/site/[uuid]/tabs/Overview.tsx (hunk 4 above)
Comment (egrojMonroy) [reply src/pages/site/[uuid]/tabs/Overview.tsx:271]: done

File: src/pages/site/[uuid]/tabs/Overview.tsx (hunk 5 above)
Comment (egrojMonroy) [reply src/pages/site/[uuid]/tabs/Overview.tsx:286]: removed

**Review by roguenet — APPROVED** [review]

**Review by roguenet — APPROVED** [review]

**Pull request merged**